pkg/controllers/disruption/orchestration/queue.go

```diff
@@ -57,6 +57,7 @@ type Command struct {
 	Replacements      []Replacement
 	candidates        []*state.StateNode
 	timeAdded         time.Time // timeAdded is used to track timeouts
+	uid               types.UID // used for log tracking
```

Comment:
```suggestion
	id               types.UID // used for log tracking
```

Is this just an `id` or `commandID` and the fact that it's a UID is an implementation detail?

Reply:
I thought commandID would stutter, since it would be command.commandID. I'm happy to do id though